.travis.yml

```diff
@@ -1,12 +1,17 @@
 language: rust
 cache: cargo
-rust:
-  - stable
-  - beta
-  - nightly
-#matrix:
-#  allow_failures:
-#    - rust: nightly
 script:
   - cargo build --verbose --all
   - cargo test --verbose --all
+matrix:
+  include:
+    - rust: stable
+    - rust: beta
+    - rust: nightly
```

Comment:
duplicating this line and adding a env variable `TEST_NODEJS_BINDINGS` would be enough and would show in the travis-ci board